src/module/vue/components/resource-meter/momentum-meter.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <AttrSlider
+    class="momentum-meter"
+    attr="momentum"
+    documentType="Actor"
+    :labelPosition="labelPosition"
+    :sliderStyle="props.sliderStyle"
+    :current-value="actor?.data.momentum ?? 2"
+    :min="-6"
+    :max="10"
+    :softMax="actor?.data.momentumMax"
+    :segmentClass="{
+      [actor.data.momentumReset]: 'segment-momentum-reset',
+    }"
+  >
+    <template #label>
+      <BtnMomentumburn
+        class="text"
+        :class="{ vertical: sliderStyle === 'vertical' }"
+        :tooltip="
+          $t('IRONSWORN.BurnMomentumAndResetTo', {
+            value: actor?.data.momentum,
+            resetValue: actor?.data.momentumReset,
+          })
+        "
+      >
+        {{ $t('IRONSWORN.Momentum') }}
+      </BtnMomentumburn>
+    </template>
+    <template #default>
+      <section class="momentum-status flexcol">
+        <span class="momentum-status-reset">
+          {{ $t('IRONSWORN.Reset') }}: {{ actor?.data.momentumReset }}
+        </span>
+        <span class="momentum-status-max">
+          {{ $t('IRONSWORN.Max') }}:
+          {{ actor?.data.momentumMax }}
+        </span>
```

Comment:
Maybe now is when can trim this. We already indicate the max by disabling the high numbers, so this is just left over from the paper sheet.

Reply:
huh, yeah, the same thing crossed my mind. we might even take it a step further by marking the reset value with an icon of some kind, and ditching that label, too. it's also conveyed in the burn button tooltip.